api/tests/routes/backoffice_v3/accounts_test.py

```diff
@@ -210,6 +210,23 @@ def test_can_search_public_account_order_by_priority(self, authenticated_client)
         assert " Théodule Dorantissime " in cards_text[1]
         assert " Théos Doranta " in cards_text[2]
 
+    @pytest.mark.parametrize(
+        "query,expected_results", [("01/01/2000", 1), ("01/01/2001", 1), ("12/12/2001", 0), ("31/31/3131", 0)]
+    )
+    def test_can_search_public_account_by_birthdate(self, authenticated_client, query, expected_results):
+        # given
+        create_bunch_of_accounts()
+        users_factories.BeneficiaryGrant18Factory(validatedBirthDate=datetime.datetime(year=2000, month=1, day=1))
+        users_factories.BeneficiaryGrant18Factory(validatedBirthDate=datetime.datetime(year=2001, month=1, day=1))
```

Comment:
```suggestion
        users_factories.BeneficiaryGrant18Factory(validatedBirthDate=datetime.date(year=2000, month=1, day=1))
        users_factories.BeneficiaryGrant18Factory(validatedBirthDate=datetime.date(year=2001, month=1, day=1))
```
si ce n'est pas une `datetime` dans le modèle